esmvaltool/cmorizers/data/downloaders/datasets/mobo_dic2004_2019.py

```diff
@@ -0,0 +1,39 @@
+"""Script to download MOBO-DIC2004-2019."""
+import logging
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+
+logger = logging.getLogger(__name__)
+
+
+def download_dataset(config, dataset, dataset_info, start_date, end_date,
+                     overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    dataset_info : dict
+         Dataset information from the datasets.yml file
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
```

Comment:
```suggestion
```

Reply:
let's try and have these functions a bit cleaner - I would add functionality for start/end times if it was me but if there's none then let's keep only the arguments we need (the downloader has them defaulted to None anyway)

Reply:
I am 99% sure that wont work since the downloader is called with these arguments:

https://github.com/ESMValGroup/ESMValTool/blob/f88b0afa045a83082704c028416d375b6b1c78a5/esmvaltool/cmorizers/data/cmorizer.py#L169-L171

Will try later though.

Reply:
cheers, Manu! If it doesn't then we should change the downloader have those optional - plopping arguments around that are not used is not kosher :fish:

Reply:
I agree that this would be cleaner, but if we want to do this properly we need to update ALL CMORizer functions and make the call to `download_dataset` dependent on the arguments of the function.

This is certainly out of scope of this PR.

Reply:
oh absolutely! (on both counts) :beer: